src/pythermalcomfort/jos3_functions/parameters.py

```diff
@@ -115,7 +115,7 @@
         "suffix": "Body name",
         "unit": "W",
     },
-    "met_base_fat": {
+    "Q_bmr_fat": {
```

Comment:
would it be possible to avoid using capital letters in the name of the variables? I have noticed that some of them are capitalised and some are not, it would be better to have them all in this format `xx_yy_zz` (we may not need to have two dashed, it was just to show the formatting)

Reply:
I tried to separate heat gain and heat loss: I use _Q_ as heat production inside the body such as basal metabolism, work, non-shivering or shivering (heat gain) and _q_ as heat transfer (basically heat loss since body temperatures are higher than the ambient) such as convective heat transfer, radiant heat transfer and so on. I am not sure this is the right way to describe, and welcome to get the suggestion. What do you think?

Reply:
now I see the point, however, since the q can also be an heat gain if we pass tdb > 35 then I would use all lower case so it is more pythonic, more consistent for the user, and it does not a priory assume heat gain or losses since they may vary. If you, however, you feel very strongly about keeping it as it is I am okay with that as well. It is just a small modification.

Reply:
I don't have strong preferences and tried to fix this, but I realized that we call total total heat loss from the skin (sensible and latent heat loss from the skin) as _q_skin_ which is the same variable name as other pythermalcomfort functions and call skin total heat production as _Q_skin_... So, I am thinking of just leaving this as it is.

Reply:
So basically we have the same name for two variables that represent two different heat loss/gains and the only difference is the capitalization? 

If this is the case I believe we should come up with unique names anyway otherwise this is very confusing. sensible and latent heat loss from the skin could be q_skil_lat_sens or q_skin_ev_cond while skin total heat production could be q_skin_gen. These are just suggestions but it is critical that we have two separate names otherwise other users and developers may struggle with this in the future. Relaying on capitalization alone is not the best strategy in my opinion.

Reply:
Only the skin layer was in conflict because of heat exchange and heat production with the surrounding environment. For the other layers it is fine. 

As shown in the commit below, 
- Q_core, Q_muscle, and Q_skin were modified as q_thermogenesis_core, q_thermogenesis_muscle, and q_thermogenesis_skin. 
- Heat loss from the skin was changed to q_skin2env. 
- All symbols was modified to lowercase, and reorderd the output parametes of JOS3 model.